src/Presence/apple.json

```
@@ -0,0 +1,282 @@
+        "message": "Notes",
+        "description": "iCloud notes"
+    },
+    "apple.iCloudReminders": {
+        "message": "Reminders",
+        "description": "iCloud reminders"
+    },
+    "apple.iCloudPagesCreate": {
+        "message": "Creating...",
+        "description": "Creating a new Pages document"
+    },
+    "apple.iCloudKeynoteWait": {
+        "message": "Waiting room...",
+        "description": ""
+    },
+    "apple.devNew": {
+        "message": "What's new",
+        "description": "New things"
+    },
+    "apple.devSubmit": {
+        "message": "Submissions",
+        "description": "Submit iOS and iPadOS apps to the App Store"
+    },
+    "apple.devPhonePad": {
+        "message": "iPhone and iPad apps",
+        "description": ""
+    },
+    "apple.devDistribution": {
+        "message": "Distributing software",
+        "description": ""
+    },
+    "apple.devDocs": {
+        "message": "Documentation",
+        "description": "Developer documentation"
+    },
+    "apple.devHIG": {
+        "message": "Human Interface Guidelines",
+        "description": ""
+    },
+    "apple.devResources": {
+        "message": "Resources",
+        "description": "Apple Design Resources"
+    },
+    "apple.devFeatures": {
+        "message": "Features",
+        "description": ""
+    },
+    "apple.devGames": {
+        "message": "Games",
+        "description": "Games like Minecraft etc."
+    },
+    "apple.forumThread": {
+        "message": "Forum - Thread",
+        "description": ""
+    },
+    "apple.forumTags": {
+        "message": "Forum - Tags",
+        "description": ""
+    },
+    "apple.forumProfile": {
+        "message": "Forum - Profile",
+        "description": "User profile"
+    },
+    "apple.forumPreferences": {
+        "message": "Preferences",
+        "description": ""
+    },
+    "apple.forumCreateThread": {
+        "message": "Creating thread",
+        "description": ""
+    },
+    "apple.forumRegister": {
+        "message": "Registration",
+        "description": "Register a new account"
+    },
+    "apple.devVideos": {
+        "message": "Videos",
+        "description": ""
+    },
+    "apple.devVidTopic": {
+        "message": "Videos - Topic",
+        "description": ""
+    },
+    "apple.devNews": {
+        "message": "News and Updates",
+        "description": ""
+    },
+    "apple.devReleases": {
+        "message": "Releases",
+        "description": "New iOS, iPadOS etc. releases"
+    },
+    "apple.btnViewProduct": {
+      "message": "View Product",
+      "description": "Button text"
+    },
+    "apple.btnViewService": {
+      "message": "View Service",
+      "description": "Button text"
+    },
+    "apple.btnViewArticle": {
+      "message": "View Article",
+      "description": "Button text"
+    },
+    "apple.btnViewEvent": {
+      "message": "View Event",
+      "description": "Button text"
+    },
+    "apple.btnViewOS": {
+      "message": "View %os%",
```

Should follow the {x} variable scheme
```suggestion
      "message": "View info about {0}",
```